src/upchunk.ts

```diff
@@ -268,6 +366,8 @@ export class UpChunk {
   private eventTarget: EventTarget<Record<EventName, UpchunkEvent>>;
 
   constructor(options: UpChunkOptions) {
+    this.eventTarget = new EventTarget();
```

Comment:
moved this up top so it's ready to go for any potential code, including any refactors to constructor setup